about.html

```diff
@@ -14,46 +14,81 @@
 <body>
     <div class="container about-page">
      
-    About
+    <div class="menu-content">
+    
+	<img class="logo" src="img/lambda-black.png" alt="Lambda Top Left Logo!">
+
+	<div class="menu-text">
+
+		<a href="index.html">Home</a>
+		<a>About</a>
+		<a>Products</a>
+		<a>Blog</a>
+		<a>Contact</a>	
+	 </div>
+	</div>	
+
+
+	<img class="about-jumbo" src="img/jumbo-about.png" alt="Jumbo for about page!">
+
+
+	<div class="about-top-content">
+    <h2>About</h2><br>
 
     Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+	</div>
+
 
+	<div class="body-content">
 
+	<div id="spacer"></div>
+
+	<div class="body-block">
     <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
+
+    <h2>Strategy</h2><br>
```

Comment:
Don't think you need the line breaks here. Headers are block elements so they automatically take up a full width by themselves and force a break. (unless you specified otherwise by changing the styling 😱)